src/Impostor.Plugins.Example/Handlers/AnnouncementsListener.cs

```diff
@@ -0,0 +1,24 @@
+using Impostor.Api.Events;
+using Impostor.Api.Events.Announcements;
+using Impostor.Api.Innersloth;
+
+namespace Impostor.Plugins.Example.Handlers
+{
+    public class AnnouncementsListener : IEventListener
+    {
+        private const int Id = 50;
```

Comment:
Id of what, what is 50, why is it 50

Reply:
No idea actually~~, I would probably know if you asked me 3 months ago when pr was created~~, probably just a random number, its a example project anyways (https://wiki.weewoo.net/wiki/Announcements)

Reply:
![image](https://user-images.githubusercontent.com/4643257/109563813-8ce4cd00-7ae0-11eb-9cdf-f2dc4b0043cf.png)

Reply:
It's sent by the client, to tell the server whether to just use the cached announcement or send a new one. The number is arbitrary, but must be customizable by config, to prevent multiple servers from having potential id collisions.

[The ID is very important. Each new announcement must have a different ID. If the ID is the same as the one from the previous announcement, it will not be displayed. ](https://wiki.weewoo.net/wiki/Announcements#1_-_Data)